_pages/documentation/gem5-stdlib/8-extending-disk-images.md

```diff
@@ -0,0 +1,119 @@
+---
+
+layout: documentation  
+title: Extending Pre-Built Disk Images Using Packer  
+parent: gem5-standard-library  
+doc: gem5 documentation  
+permalink: /documentation/gem5-stdlib/extending-disk-images  
+author: Harshil Patel  
+
+---
+
+gem5 Resources provides pre-built generic Ubuntu 24.04 disk images that you can extend and modify using Packer.
+
+### Steps
+
+1. **Download the Base Disk Image**
+
+   Obtain the base disk image from gem5 Resources. Links to the base images are available for [x86](https://resources.gem5.org/resources/x86-ubuntu-24.04-img), [ARM](https://resources.gem5.org/resources/arm-ubuntu-24.04-img), and [RISC-V](https://resources.gem5.org/resources/riscv-ubuntu-24.04-img). Download the image from the `versions` tab.
+
+2. **Unzip the Downloaded Image**
+
+   Unzip the downloaded image using `gunzip`:
+
+   ```sh
+   gunzip /path/to/image
+   ```
+
+3. **Calculate the `sha256sum` of the Unzipped Image**
+
+   Calculate the `sha256sum` of the unzipped image:
+
+   ```sh
+   sha256sum /path/to/unzipped/image
+   ```
+
+4. **Create a Packer Script**
+
+   Write a Packer script to use the above disk image as a base image and extend it. Below is an example Packer script with placeholders:
```

Comment:
Once the new npb/gapbs disk images are merged, it could be helpful to point toward the Packer scripts for those as well. It would give people more examples to look at.

Reply:
It would also be good to add a link to the directories for the base Ubuntu x86/Arm disk images to provide more examples of Packer scripts.

You could also tell readers to simply use those directories as more of the process is automated, making it easier to make disk images.